packages/pino-log-enricher/lib/createFormatter.js

```diff
@@ -39,6 +39,18 @@ module.exports = function createFormatter(newrelic) {
         }
         return obj
       }
+    },
+    hooks: {
+      logMethod(inputArgs, method, level) {
+        // We'll try to use level labels for the metric name, but if
+        // they don't exist, we'll default back to the level number.
+        const levelLabel = this.levels.labels[level] || level
+        newrelic.shim.agent.metrics.getOrCreateMetric('Logging/lines').incrementCallCount()
+        newrelic.shim.agent.metrics
```

Comment:
this just occurred to me too from the spec `|`application_logging.metrics.enabled`|`NEW_RELIC_APPLICATION_LOGGING_METRICS_ENABLED`|true|Toggles whether the agent gathers *Logging Metrics*.|`

i think we have to check config to make sure metrics for logging is enabled.  we also missed this on the winston ticket as it was already merged

Reply:
Ah, okay, I have added that check.